app/controllers/controllershelpers/ControllerLikeHelpers.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package controllers.helpers
+package controllers.controllershelpers
 
```

Comment:
If you delete:

`implicit def hc(implicit request: Request[_]): HeaderCarrier =
    HeaderCarrierConverter.fromHeadersAndSession(request.headers, Some(request.session))	    HeaderCarrierConverter.fromHeadersAndSession(request.headers, Some(request.session))`

You can remove the lines:
`implicit val hc = HeaderCarrierConverter.fromHeadersAndSession(request.headers, Some(request.session))`

The reason for this is in the `FrontendController` it does this implicit conversion already, which leads to an ambiguous implicit conversion for HeaderCarriers.